<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CConnectionBar</class>
 <widget class="QFrame" name="CConnectionBar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1140</width>
    <height>40</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>40</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>40</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::StyledPanel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>5</number>
   </property>
   <property name="leftMargin">
    <number>5</number>
   </property>
   <property name="topMargin">
    <number>5</number>
   </property>
   <property name="rightMargin">
    <number>5</number>
   </property>
   <property name="bottomMargin">
    <number>5</number>
   </property>
   <item>
    <widget class="QComboBox" name="comboConfigurations">
     <property name="minimumSize">
      <size>
       <width>150</width>
       <height>0</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="CPortsComboBox" name="comboPorts">
     <property name="minimumSize">
      <size>
       <width>250</width>
       <height>0</height>
      </size>
     </property>
     <property name="toolTip">
      <string>Select port</string>
     </property>
     <property name="editable">
      <bool>false</bool>
     </property>
     <property name="sizeAdjustPolicy">
      <enum>QComboBox::AdjustToContents</enum>
     </property>
     <property name="frame">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="comboBaudRates">
     <property name="toolTip">
      <string>Select baud rate</string>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="currentText">
      <string notr="true"/>
     </property>
     <property name="sizeAdjustPolicy">
      <enum>QComboBox::AdjustToContents</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="comboDataBits">
     <property name="toolTip">
      <string>Select data bits</string>
     </property>
     <property name="currentText">
      <string notr="true"/>
     </property>
     <property name="sizeAdjustPolicy">
      <enum>QComboBox::AdjustToContents</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="comboParity">
     <property name="toolTip">
      <string>Select parity</string>
     </property>
     <property name="sizeAdjustPolicy">
      <enum>QComboBox::AdjustToContents</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="comboStopBits">
     <property name="toolTip">
      <string>Select stop bits</string>
     </property>
     <property name="sizeAdjustPolicy">
      <enum>QComboBox::AdjustToContents</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="comboFlowControl">
     <property name="toolTip">
      <string>Select flow control</string>
     </property>
     <property name="currentText">
      <string notr="true"/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btnMore">
     <property name="text">
      <string>&amp;More &gt;&gt;</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btnConnect">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>&amp;Connect</string>
     </property>
     <property name="default">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btnSave">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>&amp;Save...</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QPushButton" name="btnClose">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../../superterm.qrc">
       <normaloff>:/icons/close-menu_16x16.png</normaloff>:/icons/close-menu_16x16.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>16</width>
       <height>16</height>
      </size>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CPortsComboBox</class>
   <extends>QComboBox</extends>
   <header>src/ui/portscombobox.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../superterm.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>btnMore</sender>
   <signal>clicked()</signal>
   <receiver>CConnectionBar</receiver>
   <slot>onMoreClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>846</x>
     <y>15</y>
    </hint>
    <hint type="destinationlabel">
     <x>665</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>comboPorts</sender>
   <signal>currentIndexChanged(QString)</signal>
   <receiver>CConnectionBar</receiver>
   <slot>onComboChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>279</x>
     <y>15</y>
    </hint>
    <hint type="destinationlabel">
     <x>665</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnClose</sender>
   <signal>clicked()</signal>
   <receiver>CConnectionBar</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1119</x>
     <y>19</y>
    </hint>
    <hint type="destinationlabel">
     <x>569</x>
     <y>19</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>comboBaudRates</sender>
   <signal>currentIndexChanged(QString)</signal>
   <receiver>CConnectionBar</receiver>
   <slot>onComboChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>452</x>
     <y>19</y>
    </hint>
    <hint type="destinationlabel">
     <x>569</x>
     <y>19</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <signal>connectClicked()</signal>
  <slot>onMoreClicked()</slot>
  <slot>onComboChanged()</slot>
  <slot>onConfigurationChanged(QString)</slot>
 </slots>
</ui>
